client/components/library.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { useState, useEffect, Fragment } from 'react';
+import EmojiCard from './emojiCard';
+import LoadButton from '../components/load-button.jsx';
+
+export default function Library() {
+  const [emojis, setEmojis] = useState([]);
+  const [emojiIndex, setEmojiIndex] = useState(8);
+  const basePath = 'api/emojis';
+  useEffect(() => {
+    fetch(basePath, {
+      method: 'GET'
+    })
+      .then(res => res.json())
+      .then(data => {
+        setEmojis(data);
+      })
+      .catch(error => console.error(error));
+  }, []);
+
+  const library = emojis.map((emoji, index) => {
+    if (index >= emojiIndex) return <Fragment key={emoji.emojiId}> </Fragment>;
```

Comment:
How is it possible for `index` to be greater than `emojiIndex`?

Maybe you could check `emojiIndex` against `emojis.length` when the emojis are read, and if necessary set `emojiIndex` to not be too large?